### DISCUSSION TITLE
For SCTP multihoming , does loxilb need to be started in HA master slave mode ?

### DISCUSSION THREAD

**Discussion by megarajan** [loxilb-io/loxilb#908] [General]
For SCTP multihoming , does loxilb need to be started in HA master slave mode ?

I am seeing the validation3.sh script in sctp-mh folder. It creates 2 llb instances llb1 and llb2 .
is HA mode mandatory for sctp multi homing ?

**Comment by nik-netlox** [comment]
Hi @megarajan ,
No, it is not a requirement. sctpmh folder contains test suites for sctp multihoming feature. Topology was created so that it can be used for HA cases as well. validation3.sh scripts tests sctp multihoming with connection sync in HA environment.

**Reply by megarajan** [reply]
thanks for clarification ..
